app/views/payments/ChargeTypeDetailsView.scala.html

```diff
@@ -0,0 +1,130 @@
+                href = Some(appConfig.btaHomeUrl)
+            ),
+            BreadcrumbsItem(
+                content = Text(messages("vatDetails.title")),
+                href = Some(controllers.routes.VatDetailsController.details.url)
+            ),
+            BreadcrumbsItem(
+                content = Text(messages("openPayments.title")),
+            )
+        )
+    ))
+}
+
+@dueDateHtml = {
+    @model.dueDate
+}
+
+@chargeDueHtml = {
+    @model.originalAmount
+}
+
+@clearedAmountHtml = {
+    @model.clearedAmount.getOrElse(0)
+}
+
+@outstandingAmountHtml = {
+    @model.outstandingAmount
+}
+
+@mainTemplate(
+    title = model.chargeTitle,
+    appConfig = appConfig,
+    serviceInfoContent = serviceInfoContent,
+    user = Some(user),
+    navLinkContent = Some(breadcrumbs)
+) {
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            @if(model.periodFrom && model.periodTo) {
+                <span class="govuk-caption-xl">
+                    @displayDateRange(model.periodFrom.get, model.periodTo.get, false, true)
+                </span>
+            }
+            <h1 class="govuk-heading-xl">
+                @model.chargeTitle
+            </h1>
+
+            @govukSummaryList(SummaryList(
+                rows = Seq(
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("chargeTypeDetails.dueDate"))
+                        ),
+                        value = Value(
+                            content = HtmlContent(dueDateHtml)
+                        )
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("chargeTypeDetails.chargeDue"))
+                        ),
+                        value = Value(
+                            content = HtmlContent(chargeDueHtml)
+                        )
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("chargeTypeDetails.amountReceived"))
+                        ),
+                        value = Value(
+                            content = HtmlContent(clearedAmountHtml)
+                        )
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("chargeTypeDetails.amountLeft"))
+                        ),
+                        value = Value(
+                            content = HtmlContent(outstandingAmountHtml)
+                        )
+                    )
+                )
+            ))
+        </div>
+    </div>
+
+    @govukButton(Button(
+        content = Text(messages("chargeTypeDetails.button")),
+        href    = Some(appConfig.unauthenticatedPaymentsUrl)
+    ))
+
+    <a class="govuk-link" href="#"
```

Comment:
When they aren't wrapped in something else, Govuk links render quite small, and in the screenshot the link looks "standard" size to me. Hard to say for sure without seeing the page here. But I'd play safe and wrap this in a `<p class="govuk-body">`